app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -0,0 +1,49 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_response")
+public class SenderSyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
+	private String messageUuid;
+	
+	@NotNull
+	@Column(name = "date_sent", nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
```

Comment:
The value of this date field is parsed from the payload sent by the receiver, I suggest you change this type from Date to LocalDateTime so that you don't have to add and use the new method in DateUtils

Reply:
Changed accordingly.

Reply:
The field has been changed to dateSentByReceiver, as we discussed.